### ISSUE TITLE
Latest Visual Studio 2022. 17.4.5  seems to have broken Indicators

### ISSUE THREAD

**Issue by gabsamples6** [AndreiMisiukevich/CardView.MAUI#3]
Hi,
I hope you have time to work out whats going on but it seems that the indicators no longer work or they are totally stretched in the latest version of visual studio  see pic below taken from you sample app.  Now all the pages where we use the indicators are looking  really bad. Most likely a Maui bug but may be you can suggest a workaround.  thanks for this beatiful control 

![image](https://user-images.githubusercontent.com/109357537/219570309-d065ab85-259a-4942-9ef1-a8214a9e7a77.png)

**Comment by gabsamples6** [comment]
@AndreiMisiukevich  hi I seem to have found a workaround but the indicators are not centered
all I did is set the Height and Widthrequest and did it but does not center I hate to use negative margins..
Could it be because is a stacklayout ? 

        <controls:IndicatorsControl
            HeightRequest="20"
            ToFadeDuration="1500"
            WidthRequest="20" />
        <controls:LeftArrowControl ToFadeDuration="2500" />
        <controls:RightArrowControl ToFadeDuration="2500" />

![image](https://user-images.githubusercontent.com/109357537/219588511-d30dba7c-ee19-4589-8e44-d4f1d9574adf.png)

**Comment by AndreiMisiukevich** [comment]
Hi @gabsamples6
Yeah :( Unfortunately, MAUI is too buggy for now in comparison to Xamarin.Forms.

Honestly, I think it's an MAUI issue since it worked fine on XF.

Unfortunately, I don't want to add dozens and dozens of workarounds and tricks to the code base because the toolkit is pretty raw so far.

**Comment by SarthakB26** [comment]
Yes workaround due to MAUI bugs will make code more buggy eventually. So we should wait I guess

**Comment by AndreiMisiukevich** [comment]
pls check v0.0.4
